src/sections.rs

```diff
@@ -142,6 +147,10 @@ macro_rules! define_custom_sections {
 define_custom_sections! {
     Name(Vec<NameSubSection>) = "name",
     Producers(Vec<ProducerField>) = "producers",
+    // https://github.com/WebAssembly/tool-conventions/blob/08bacbed/Debugging.md#external-dwarf
+    ExternalDebugInfo(ExternalUrlSection) = "external_debug_info",
```

Comment:
I think you can skip the custom struct and just use `String` here directly.

Reply:
I don't feel too strongly about it though, just saying as it seems simpler.

Reply:
The first suggestion seems to work nicely, I couldn't get the second syntax working though, I'm probably missing something.

I don't feel strongly either, the documentation of saying this URL is a string seems somewhat nice 🤷